libs/types/src/cast_vote_records.ts

```diff
@@ -20,3 +21,14 @@ export interface CastVoteRecord
   readonly _testBallot: boolean;
   readonly _scannerId: string;
 }
+
+/**
+ * Metadata stored in the top-level metadata file for a cast vote record export
+ */
+export interface CastVoteRecordExportMetadata {
+  arePollsClosed?: boolean;
+  /** Global data relevant to all cast vote records in an export, e.g. election info */
+  castVoteRecordReportMetadata: CastVoteRecordReport;
+  /** A hash of all cast vote record files in an export */
+  castVoteRecordRootHash: string;
```

Comment:
I would guess you address this later or somewhere else, but right now not sure why the root hash exists alongside the report metadata rather than entirely separately, because I would think the root hash should include the report metadata.

Reply:
Good question! This hash doesn't need to contain the metadata file itself because the metadata file is what's we're signing with OpenSSL. So everything in it is inherently incorporated into the signature.

Side note: When you ask OpenSSL to sign something, it actually hashes whatever you pass to it and then signs that hash. That's why we don't have to pre-hash the metadata file, itself.

That said, I do see why this might cause confusion. I'll think some more about whether it's worth changing in subsequent PRs. One nice property of the current setup is that we can trust the metadata file after a very speedy signature check. If we were to incorporate the metadata file alongside all the other files into the root hash and just sign that hash, we'd have to recompute the root hash (a slower operation) before we could trust the metadata file. There might be some convenience in being able to trust the metadata file more quickly, ahead of the rest of the files in the export.

Getting a bit in the weeds here but we can chat further about this during our sync tomorrow!

Reply:
That explains it. Not everything needs to be hashed - everything just needs to be signed.

Reply:
> Not everything needs to be hashed

^ Exactly! More specifically, not everything needs to be hashed by our code, pre-OpenSSL

Ultimately, even the metadata file is hashed because that's what OpenSSL always does before signing (following the general best practices around signatures)